operation/fhir-operation-healthcheck/src/main/java/com/ibm/fhir/operation/healthcheck/HealthcheckOperation.java

```diff
@@ -45,9 +49,22 @@ protected Parameters doInvoke(FHIROperationContext operationContext, Class<? ext
         try {
             FHIRPersistence pl =
                     (FHIRPersistence) operationContext.getProperty(FHIROperationContext.PROPNAME_PERSISTENCE_IMPL);
-            OperationOutcome operationOutcome = pl.getHealth();
-            checkOperationOutcome(operationOutcome);
-            return FHIROperationUtil.getOutputParameters(operationOutcome);
+
+            FHIRPersistenceTransaction tx = resourceHelper.getTransaction();
+            tx.begin();
+            
+            try {
+                logger.info("checking health");
```

Comment:
Please remove this. 
It gets called every 60 seconds...

Reply:
this was removed a while back